DESCRIPTION

```diff
@@ -0,0 +1,31 @@
+Package: vivax
+Type: Package
+Title: An R wrapper for a P. Vivax malaria transmission model
+Version: 0.0.1
+Date: 2020-01-31
+Authors@R: c(
+  person(
+    given = "Michael",
+    family = "White",
+    role = c('aut', 'cre')
```

Comment:
Somewhat counter intuitively, `cre` needs to go with the *maintainer* not really the creator. And because Michael does not have an email address you're getting the build error (though by the time this review is submitted you will likely have worked that out :) )